lib/charms/vault_k8s/v0/vault_kv.py

```diff
@@ -367,7 +369,7 @@ def remove_unit_credentials(self, relation: ops.Relation, nonce: Union[str, Iter
 
         relation.data[self.charm.app]["credentials"] = json.dumps(credentials, sort_keys=True)
 
-    def get_credentials(self, relation: ops.Relation) -> Mapping[str, str]:
+    def get_credentials(self, relation: ops.Relation) -> Any:
```

Comment:
what was the issue with the Mapping?

Reply:
`json.loads()` won't return a `Mapping`, so it was complaining. I believe a `Mapping` is immutable.

`json.loads()` returns `Any`, although for an object like this it will always convert it to a `dict`. We should be safe to assume a `dict`, since pydantic will enforce that on the way in. 

I'll change this to `dict`.